tests/console/validate_raid_encrypt_home.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Simple RAID partitioning layout validation for encrypted home
```

Comment:
Add test steps will be better.

Reply:
I haven't updated here because haven't a better summary for the test module, with that then don't need test steps which will be easily understand through summary and code.

Reply:
"Validate that a RAID multi-disk is directly formatted with an encrypted home".
this would be better than any steps.